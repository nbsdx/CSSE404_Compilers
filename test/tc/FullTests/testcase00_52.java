// Invalid assignment of variable a

class bar
{
	public static void main(String[] args)
	{
		bool a = 4;
	}
}
