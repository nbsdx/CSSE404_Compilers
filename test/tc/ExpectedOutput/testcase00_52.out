Cannot assign type int to variable a of type bool
